src/kirby/components/card/card.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, OnInit, Input, ElementRef, Renderer2, OnDestroy } from '@ang
 
 import { ResizeObserverService } from '../shared/resize-observer/resize-observer.service';
 import { ResizeObserverEntry } from '../shared/resize-observer/types/resize-observer-entry';
-import { ColorHelper, ColorType } from '../../helpers/color-helper';
+import { ColorType } from '~/kirby/helpers/color-type';
```

Comment:
Don't use `~` imports as it breaks symlinking (there's as reason why the line above uses relative path...)